src/config.ts

```diff
@@ -11,7 +11,11 @@ interface Configuration {
   // The user visible version string fro the application running Glean.js.
   readonly appDisplayVersion?: string,
   // The server pings are sent to.
-  readonly serverEndpoint?: string
+  readonly serverEndpoint?: string,
+  // Whether or not Glean should be started in testing mode.
+  //
+  // This unlocks features that allows dispatcher tasks to be waited on.
+  readonly testing?: boolean
```

Comment:
```suggestion
  readonly testing?: boolean,
```

Does the ts compiler like the trailing comma? If so, we should rather use this, it makes diffs clearer.

Reply:
Honestly need to read up on this, the ts compiler doesn't care. There times in the code I used `;` for this too. Need to check if there are differences and if not, decide on a standard and add a eslint rule if possible.

Reply:
Let's stick to the trailing comma, that's what is being used in other Mozilla projects, too.